emukit/examples/preferential_batch_bayesian_optimization/__init__.py

```diff
@@ -0,0 +1 @@
+from .pbbo import *
```

Comment:
I don't think this is needed. What is needed though is the __init__.py itself, but just empty

Reply:
You are right, fixed!